openeogeotrellis/job_registry.py

```diff
@@ -174,6 +177,10 @@ def mark_done(self, job_id: str, user_id: str) -> None:
 
         self._zk.delete(source, version)
 
+        _log.debug(
+            f"marked {job_id=} as done", extra={"job_id": job_id, "user_id": user_id}
+        )
```

Comment:
aha that is interesting

my attempt was to generalize this log:
 https://github.com/Open-EO/openeo-geopyspark-driver/blob/68bf71e2991657145cd76e3f8ec27700642bb630/openeogeotrellis/job_tracker.py#L135

but apparently it has a side effect.

In what sense will things be messed up? that too much or wrong "marked done" events will be captured, or stopped being captured? Or is is something about the log format and the "extra" fields?

Reply:
I was thinking along the lines of: that processor might pick up those extra logs and downstream processors might choke on them because they're missing a `unique_process_ids` extra. But yeah, this is to be looked into.